drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -32,6 +32,8 @@ LOG_MODULE_REGISTER(usb_nrfx);
 /* USB device controller access from devicetree */
 #define DT_DRV_COMPAT nordic_nrf_usbd
 
+#define USB_REQTYPE_GET_TYPE(bmRequestType) (((bmRequestType) >> 5) & 0x03)
```

Comment:
Disagree. I thought we are going in direction of unifying all macros that could be used across the project or remove them at all.